### PR TITLE
State of Mozilla 2019-2020

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Description

2019-2020 State of Mozilla Page

## Issue / Bugzilla link

#9727

## To-do

- [x] _Non-blocking:_ Remove Modal destory/create animation during next/prev calls (@maxxcrawford)
- [x] Confirm opengraph page sharing looks good (@craigcook)
- [x] Browser Test (@craigcook)
- [x] Confirm mobile/responsive layout (@craigcook)
- [x] Run Integration Test Branch Test Suite (@maxxcrawford)
- [x] Confirm no [token abuse](https://protocol.mozilla.org/fundamentals/tokens.html) in SCSS (@craigcook)
- [x] Remove/Replace All Draft Copy (@maxxcrawford)
- [x] Remove all unused/2019 stub copy (@maxxcrawford)
- [x] Remove unused SCSS (from this commit 94bf37962d2ad3bb7f7173b300acfb49a5534545)  (@maxxcrawford)
- [x] Add all images (modal too) (@craigcook) 
- [x] Confirm images are lazy loaded (@craigcook))
- [x] Add leadership letters (@maxxcrawford)
- [x] Add "Prev/Back" Functionality to Modal (@maxxcrawford)
- [x] Put the page behind a switch

## Testing

- https://www-demo-pr-9711.herokuapp.com/en-US/foundation/annualreport/2019/
- https://www-dev.allizom.org/en-US/foundation/annualreport/2019/

### CODE REVIEW

File: media/css/foundation/annual-report.scss [hunk 1]
```diff
@@ -332,3 +347,22 @@
         overflow-x: visible;
     }
 }
+
+.mzp-l-card-half.is-reverse {
+    .mzp-c-card {
+        @media #{$mq-sm} {
+            float: right;
+            margin-right: 0;
+            margin-left: $spacing-xl;
```
Comment (stephaniehobson) [inline media/css/foundation/annual-report.scss:356]: I wish I had some docs to point you to but I couldn't find any 🤦‍♀️  We usually use the `bidi` mixin for RTL stuff:

```suggestion
        @include bidi((
            (float, right, left),
            (margin-right, 0, $spacing-xl),
            (margin-left, $spacing-xl, 0),
        ));
```

Comment (craigcook) [reply media/css/foundation/annual-report.scss:356]: bidi docs! https://protocol.mozilla.org/docs/contributing.html#bidi

**Comment by craigcook** [comment]
@maxxcrawford I've added all the images and got them lazyloading. 

I saw you were serving all the past reports from the same base style sheet (fair enough considering this started as a rough draft). But now that the 2019 report is a bespoke page so it didn't seem worth loading all that custom styling on pages that don't use it, so I split things into two bundles. I also removed the old draft content and did some other simple fixes along the way as I spotted them, things like fixing link URLs, adding UTM params to external links, etc.

Plenty more we could do but I'm feeling pretty good about finishing this up on Friday.

**Review by craigcook — APPROVED** [review]
Although there's plenty of room for improvement, the rushed nature of this work has necessitated cutting some corners and letting a lot of things slide. Quick and dirty is what we get when content isn't provided until the last minute. So all things considered, amazing work! r+ 📆

**Pull request merged**